### DISCUSSION TITLE
Im getting an error related to Lora folder

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#1794] [Q&A]
Can you help me with this error? idk what to do :(

Every time I try to run SD in colab, i get this:

reading lora /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Lora/yamamuraSadakoTheRing_v3.safetensors: OSError Traceback (most recent call last): File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions-builtin/Lora/lora.py", line 53, in init File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/sd_models.py", line 216, in read_metadata_from_safetensors OSError: [Errno 107] Transport endpoint is not connected: '/content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Lora/yamamuraSadakoTheRing_v3.safetensors' reading lora /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Lora/youreUNDERARRESTLoha_v10.safetensors: OSError Traceback (most recent call last): File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions-builtin/Lora/lora.py", line 53, in init File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/sd_models.py", line 216, in read_metadata_from_safetensors OSError: [Errno 107] Transport endpoint is not connected: '/content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Lora/youreUNDERARRESTLoha_v10.safetensors' Error loading script: lora_script.py Traceback (most recent call last): File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/scripts.py", line 248, in load_scripts File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/script_loading.py", line 11, in load_module File "<frozen importlib._bootstrap_external>", line 850, in exec_module File "<frozen importlib._bootstrap>", line 228, in callwith_frames_removed File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions-builtin/Lora/scripts/lora_script.py", line 5, in <module> File "<frozen importlib._bootstrap>", line 1007, in findand_load File "<frozen importlib._bootstrap>", line 982, in findand_load_unlocked File "<frozen importlib._bootstrap>", line 925, in findspec File "<frozen importlib._bootstrap_external>", line 1423, in find_spec File "<frozen importlib._bootstrap_external>", line 1395, in getspec File "<frozen importlib._bootstrap_external>", line 1526, in find_spec File "<frozen importlib._bootstrap_external>", line 1569, in fillcache OSError: [Errno 107] Transport endpoint is not connected: '/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions-builtin/Lora' Error loading script: scunet_model.py Traceback (most recent call last): File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/scripts.py", line 248, in load_scripts File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/script_loading.py", line 11, in load_module File "<frozen importlib._bootstrap_external>", line 846, in exec_module File "<frozen importlib._bootstrap_external>", line 982, in get_code File "<frozen importlib._bootstrap_external>", line 1039, in get_data OSError: [Errno 107] Transport endpoint is not connected: '/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions-builtin/ScuNET/scripts/scunet_model.py'

The list of errors like this goes on after this.

But, if I remove the Lora folder before running SD, I dont get the error. But I can't use my Loras then.

Any help is appreciated, thanks.

**Comment by TheLastBen** [comment]
I'll check it out

**Reply by gilroff** [reply]
I fixed the issue temporarily. I made a new folder inside the Lora folder and moved half of my loras there. Still the issue is that loading many lora files in a single folder causes crash.

**Comment by HenryFroio** [comment]
I've been having this erros too, started last week. Only way to fix is removing loras :(

**Reply by gilroff** [reply]
@HenryFroio I found a temporary fix:
Dont put loras and files directly in lora folder. put ur loras and files in folders inside lora folder.
each folder shouldn't have more than 297 total files.
i dont know the exact maximum file number, but 297 files work for me.
